src/Symfony/Component/Security/Core/Authentication/Token/AccessToken.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authentication\Token;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * AccessToken implements an access token.
+ * This type of token is commonly used in API contexts.
+ *
+ * @author Florent Morselli <florent.morselli@spomky-labs.com>
+ */
+class AccessToken extends AbstractToken
```

Comment:
Do we need a special token, or can we use the default `PostAuthenticationToken`.

Mainly asking this to @chalasr/the rest of the core team. For legacy reasons, many authenticators use a specific token (e.g. `UsernamePasswordToken`, although there is no password in there anymore). But the latest `LoginLinkAuthenticator` also just uses the default token.

I would say there is no need for a special token?

Reply:
I agree, let's not introduce unnecessary complexity again.

Reply:
OK that makes sense. I removed the AccessTokenclass and now the method returns the default `PostAuthenticationToken`